shell.client/src/main/java/org/metaborg/spoofax/shell/client/Repl.java

```diff
@@ -43,6 +44,9 @@
         this.editor = editor;
         this.display = display;
         this.invoker = invoker;
+
+        this.editor.setPrompt(new StyledText(Color.RED, "[In ]: "));
```

Comment:
What happened to my comments on #2 ?

> Small nitpick, but I (and I think others too) associate red with error, so perhaps the default color should be changed to something without an associated meaning. Also, I think we should stick with symbols only, unless we start to get more than just singleline/multiline prompts (my suggestion is to follow Python and use `>>>` for singleline and `...` for multiline)

https://github.com/spoofax-shell/spoofax-shell/pull/2#discussion_r62495221